TurtleShroom Productions III !

TurtleShroom opened the pull request.

TurtleShroom commented:
@loup99 

Here is the latest batch of works I've produced. Enjoy!

loup99 commented on WTWSMS/common/landed_titles/mercenaries.txt:
```diff
@@ -715,40 +715,6 @@ d_sunni_cuman_company = {
-	
-	mercenary_type = berber_company_composition
-}
-
```
Why did you move these mercenaries?

TurtleShroom replied:
I moved the "Knights of the VALENTINIANS" over to the Holy Orders because it's a Valentinian Holy Order.

loup99 commented on WTWSMS/decisions/VIET_Eastern_religions_decisions.txt:
```diff
@@ -31,7 +32,7 @@ decisions = {
 				}
 			}
-			NOR = {
+			NOT = {
```
Why did you change this?

TurtleShroom replied:
@loup99 

It was an error. "NOT" tells the code that if any one of those conditions are met, the action won't be applied or enabled. "NOR" is not a command. It was a typo.

loup99 replied:
NOR is a command.

TurtleShroom replied:
@loup99 

Isn't that what "NOT" does? If anything in a "NOT" zone is true, the entire thing is rejected. Is "NOR" the opposite?

loup99 replied:
NOT returns true if the enclosed condition is false.

loup99 commented on WTWSMS/decisions/WtWSMS_realm_decisions.txt:
```diff
@@ -350,6 +350,10 @@ decisions = {
+			add_law = succ_primogeniture
+			add_law = vice_royalty_2
+			add_law = imperial_administration
+			set_allow_free_vice_royalty_revokation = yes
```
This gives it freely though.

TurtleShroom replied:
I don't understand what you mean.

loup99 replied:
I don't want those who restore the roman empire to receive a free boost in terns of laws.

TurtleShroom replied:
@loup99  

Okay. That still doesn't address why the two Roman Empires don't have the Imperial Administration Law in effect at the game's start, like they would have had in RL. What I wrote about _uniting_ (not restoring) the two Roman Empires was a makeshift quick fix because I had no idea how to code the two empires to start with Imperial Administration.

I'll revert it if you'd like.

loup99 replied:
Preferably yes.

loup99 commented on WTWSMS/localisation/WtWSMS_religions.csv:
```diff
@@ -737,6 +739,15 @@ HYMNBOOK;The Black Hymnal;;;;;;;;;;x
+create_salem;Instate Salem;;;;;;;;;;;;;x
+create_salem_desc;Melchisedechianism has always been dismissed as a strange, mysticist splinter of Christianity. No more! We are now in the position to restore the ancient Kingdom of Salem, that Father Abraham himself gave his wealth to, and glorify God's name as He deserves once more! Glory to Salem, glory to El-ʿElyōn!;;;;;;;;;;;;;x
+create_melchisedechianism_holy_order;Form the Order of Melchizedek;;;;;;;;;;;;;x
+create_melchisedechianism_holy_order_desc;With the Kingdom of Salem in place, we can now begin to focus on defending the faith from the onset of heathenry and paganry that we face every day. Let us convene the most zealous, devoted, and brilliant strategists to defend the followers of Melchezedek wherever they live!;;;;;;;;;;;;;x
```
No decisions should be in this file.

TurtleShroom replied:
@loup99 

Gladly. I will fix it immediately.

loup99 commented on WTWSMS/localisation/WtWSMS_religions_unused.csv:
```diff
@@ -56,8 +56,6 @@ GOD_OMNIPATER;The Omnipater;;;;;;;;;;;x
 GOD_THE_FROGMEN;The Frogmen of Wasketville Court;;;;;;;;;;;x
 GOD_CONNOR_ANDREW_DAY;Connor Andrew Day;;;;;;;;;;;x
-banditism;Banditism;;;;;;;;;;;x
-banditism_DESC;The highly heretical Bandit sect claims that the almighty Delorius is not a unicorn, but instead a regular horse named Bandit. Definitely an Athiest sect, thanks Communist Obama!;;;;;;;;;;;x
```
I get most of it, and this file is unused anyway.

TurtleShroom replied:
Can we still remove it anyway? It irritates me that it's there.

loup99 replied:
If that irritates you out of everything, WtWSMS is not your can of works. :stuck_out_tongue:

TurtleShroom replied:
It irritates me so badly because it's an incorrect use of the "THANKS OBAMA" meme.

loup99 commented on WTWSMS/common/cb_types/WtWSMS_migration_unification_cb_types.txt:
```diff
@@ -39,12 +39,23 @@ migration = {
+	
 					NOT = { culture = FROM }
 					independent = yes # No vassals
 					war = no # Ensures that the events activate correctly  
```
I disagree on this change. Migration is limited with a purpose, and you both expand it religion-wise and disable it for all non-nomads.

TurtleShroom replied:
@loup99 

SHOOT! Loup, I'm so sorry. I didn't mean to disable it for everyone except nomads.

What I was trying to code was this:

1. Nomads can migrate, but no other type of Emperor can.
2. Reformed incarnations of the previously available nomads can still migrate. Why should a Slavic or Germanic/Norse Pagan stop doing what they do best?

For the sake of historical accuracy, I also think that Magyars, Iazygens, and Iranian Steppe Pagans should be allowed to migrate freely (as they did in RL), but I decided to save that for another time.

loup99 replied:
I think the current setup should be enough for 1 to not be an issue. On 2, that is a design question. I disagree with you and think reformed pagans are more settled than the unreformed versions.

loup99 commented on WTWSMS/common/scripted_effects/WtWSMS_scripted_effects.txt:
```diff
@@ -4,12 +4,13 @@
+			martial = 10
 		}
 		trait = skilled_tactician
 	}
```
This will be done but later.

TurtleShroom replied:
Okay. I'll remove it now.

loup99 commented on WTWSMS/decisions/minor_decisions.txt:
```diff
@@ -3,11 +3,6 @@ decisions = {
-			}
 			prisoner = no
 			OR = {
 				religion_group = christian
```
This requirement should be kept.

TurtleShroom replied:
Why can't any government meeting the religious requirements hold a feast? Are you telling me that wealthy Merchant Republics didn't pig out from time to time?

loup99 replied:
Not in the way this one works.

loup99 commented on WTWSMS/localisation/WtWSMS_religions_unused.csv:
```diff
@@ -241,5 +241,5 @@ dacian_pagan;Zalmoxian;;;;;;;;;;;x
-roman_pagan_DESC;The Romanic heresy is a deviant branch of the Hellenic church the believes the addition of Greek deities corrupted the original Roman faith. Also anti-clerical in nature, the Romanic faith has no religious head or special game mechanics.;;;;;;;;;;;x
\ No newline at end of file
+romanic_pagan;Romanic;;;;;;;;;;;x
+romanic_pagan_DESC;The Romanic heresy is a deviant branch of the Hellenic church the believes the addition of Greek deities corrupted the original Roman faith. Also anti-clerical in nature, the Romanic faith has no religious head or special game mechanics.;;;;;;;;;;;x
```
Leave this file.

loup99 commented on WTWSMS/common/minor_titles/WtWSMS_imperial_legacy_minor_titles.txt:
```diff
@@ -466,11 +492,12 @@ title_il_comes_sacrae_vestis = {
+	}
 
 	allowed_to_hold = {
 		is_adult = yes
```
The requirement should in my opinion be for Romans.

TurtleShroom replied:
I didn't change that. What I added was that the Restored Roman Empire could use ERE AND WRE titles. Everything else stayed the same (must be Roman culture, must hold RE, WRE, or ERE, etc.).

loup99 replied:
Why did you remove culture = roman?

loup99 commented on WTWSMS/localisation/ACR_hellenic_events.csv:
```diff
@@ -103,3 +103,10 @@ ancrel.2568.title;The Sacred Band;La Bande Sacr
+HAIL_SATURN2;Mastiman Must Feed!;;;;;;;;;;;;;x
+HAIL_SATURN1_DESCRIPTION;[Root.GetFOA], we have learned that the filthy Mastimanis have sacrificed [From.GetBestName] to their bloodthirsty demons! After setting up an altar and presenting food and drink offerings, [From.GetBestName] was slaughtered and [From.GetHisHer] body lay mutilated before the throngs of cheering faithful.;;;;;;;;;;;;;x
+HAIL_SATURN2_DESCRIPTION;Mastiman is hungry. Mastiman craves blood. Let the Augurs do what must be done. [From.GerBestName] should be honored to fulfill such a noble and holy goal.;;;;;;;;;;;;;x
+FEEDMASTIMAN;BLOOD FOR THE BLOOD GOD: SOULS FOR MASTIMAN! ALL HAIL!;;;;;;;;;;;;;x
```
I have already explained why this is not necessary.

TurtleShroom replied:
Darn. I was too slow.

TurtleShroom commented:
@loup99 

Okay, I reverted the Mastimani addition because it has already been added. I also reverted the "NOT" back to "NOR".

Is that what you wanted me to do?

loup99 commented:
Could you maybe redo this one? There are conflicts with master and most of it is either obsolete/undesired. I will also do compatibility, so it will just be less compatible.
